airflow/www/fab_security/manager.py

```diff
@@ -0,0 +1,1952 @@
+import base64
```

Comment:
Need to add Apache-license same as below:

https://github.com/apache/airflow/blob/d4a056647e0f4bf76d1139853917149b907beb2d/airflow/config_templates/default_webserver_config.py#L1-L17

Reply:
Thanks Kaxil, I've added the license.